### PR TITLE
reuse items

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

**Comment by jmhauck** [comment]
@MikeTschudi this previously showed as passing :( will update the tests more

File: packages/common/test/mocks/agolItems.ts [hunk 1]
```diff
@@ -708,7 +708,7 @@ export function getAGOLGroup(): any {
     owner: "LocalGovTryItLive",
     description: "Description of an AGOL group",
     snippet: "Snippet of an AGOL group",
-    typeKeywords: ["JavaScript"],
```
Comment (MikeTschudi) [inline packages/common/test/mocks/agolItems.ts]: Why move "JavaScript" from typeKeyword to tag?  For WMAs, e.g., the typeKeyword is set behind the scenes to "JavaScript" whether or not the user sets it also as a tag.

Comment (jmhauck) [reply packages/common/test/mocks/agolItems.ts]: @MikeTschudi that function gets a group. Group doesn't have typeKeywords only tags.

Comment (MikeTschudi) [reply packages/common/test/mocks/agolItems.ts]: Thanks!

**Comment by jmhauck** [comment]
#175

**Pull request merged**